hack/ci/run-konnectivity-e2e-test.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
```

Comment:
If this script can run in CI, do we want to enable it in [.prow.yaml](https://github.com/kubermatic/kubermatic/blob/69d12dbb03d6524f150be438f9c9030669d0f835/.prow.yaml) ?

Reply:
I am working on making it a cron job here https://github.com/kubermatic/infra/pull/1624/files

Reply:
OK, and in addition to cron execution, could this job also be triggered (only) on PRs that touch some specific Konnectivity packages? Like:
 - /pkg/resources/konnectivity/
 - /pkg/controller/user-cluster-controller-manager/resources/resources/konnectivity/